### PR TITLE
readme first 2 commits

### PR DESCRIPTION
**Pull request by inc0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+experience as proficient operators. We don't assume any technology they
+are using.
+
+Our OpenStack will be:
+
+* production ready - this cloud is meant to serve as production environment
+* extensible - extending cloud after initial deployment
+* upgradable - it will be as easy as possible to do release upgrades
+* easy to deploy - it will require minimal input from operators
+* quick to deploy - our goal is to get down to 1min/node on average in scale of 300 nodes
```
Comment (cloudnull) [inline README.md:31]: This is not a deployment reality, especially at scale. While its a nice goal its not something that will ever be feasible for all services in all situations and I'd rather not have call it out as a governing design decision.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+OSIC Reference implementation
+=============================
+
+# What is it?
+
+Reference implementation is opinionated way to deploy OpenStack. One of big
```
Comment (cloudnull) [inline README.md:6]: Are we forcing our opinions upon deployers or making suggestions as to how we think they should build a cloud?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+qualities of OpenStack is it's flexibility, modularity and amount of underlying
+technilogies it can work with. But that also means that it has steep learning
+curve as sheer amount of options it allows is overwhelming (some services can
+have hundreds of them). We want to ease the entry to OpenStack by providing
+what is in our opinion good way to deploy OpenStack. We make sure that it's
+possible to be consumed by majority of users because it will be based on
+opensource as much as it is reasonable. In short, it's OpenStack authors would
+deploy:)
```
Comment (cloudnull) [inline README.md:14]: How are we quantifying this statement "In short, it's OpenStack authors would deploy:)" -- I'm not sure it has a place in this document.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+# Design assumptions
+
+RefImpl will be designed to optimally run generic web services/application
+workload on a medium scale cloud (20 nodes to start with plans to expand
+to 300). Main users of it will be companies that are willing to deploy an
+OpenStack private cloud, but have no or minimum OpenStack experience, only
+experience as proficient operators. We don't assume any technology they
+are using.
```
Comment (cloudnull) [inline README.md:23]: I think for us to have an opinion on how the cloud should scale and operate we have to have an opinion on the tech they are using.

File: README.md (hunk 1 above)
Comment (inc0) [inline README.md:31]: This is one of goals set up for us by director of OSIC.

File: README.md (hunk 2 above)
Comment (inc0) [inline README.md:6]: Forcing our opinions, take it or leave it approach with caveat of howtos about overriding our decisions, but with "you are on your own" clause

File: README.md (hunk 4 above)
Comment (inc0) [inline README.md:23]: What tech that would be? We can go down to a bare metal configuration.

**Comment by inc0** [comment]
Also there are more questions unanswered. Like actual audience for it. Let's hold off this discussion for PMs to agree on goals

File: README.md (hunk 1 above)
Comment (odyssey4me) [inline README.md:31]: If this is 1m of active time (ie time taken by a human to do something) then this is fine as automation takes care of it. I think this should be re-worded to make this clearer.

**Pull request closed without merging**